src/neuroconv/datainterfaces/ecephys/requirements.txt

```diff
@@ -1,3 +1,3 @@
-spikeinterface>=0.95.1
+spikeinterface @ git+https://github.com/spikeinterface/spikeinterface@6e5604b6671df3af5c648b389c9925d96abbdfca # main branch on 12/23/22, 1:25pm EST
```

Comment:
spikeinterface just released a new version. Can't we use that latest release?

Reply:
Yes that's the point, was just about to update it